gcs-action/jest.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
The new actions are using mocha instead of jest.

Reply:
Done